amf_test.go

```diff
@@ -59,8 +59,7 @@ func TestInitialConfig(t *testing.T) {
 	factory.AmfConfig.Configuration.PlmnSupportList = nil
 	factory.AmfConfig.Configuration.ServedGumaiList = nil
 	factory.AmfConfig.Configuration.SupportTAIList = nil
-	var Rsp chan *protos.NetworkSliceResponse
-	Rsp = make(chan *protos.NetworkSliceResponse, 1)
+	var Rsp chan *protos.NetworkSliceResponse = make(chan *protos.NetworkSliceResponse, 1)
```

Comment:
What about the following for simplicity?
```suggestion
	Rsp := make(chan *protos.NetworkSliceResponse, 1)
```